distribution/pom.xml

```
@@ -371,42 +369,81 @@
               </execution>
             </executions>
             <configuration>
-              <copyright>2013 ASF</copyright>
+              <license>http://www.apache.org/licenses/LICENSE-2.0</license>
```

Looks like the following content for the license will be better, see [docs](https://github.com/mojohaus/rpm-maven-plugin/blob/9d66d4dbbbefb79a0fe01e0ef538668f8444cbcd/src/site/apt/ident-params.apt#L55):
```suggestion
              <license>${project.organization} ${project.licenses}</license>
```

---

I am not able to see 1.16.0 branch on https://github.com/apache/drill.git either "git tag" or "git branch -a" command will list the 1.16.0 branch. Can you please suggest ?

---

@vdiravka I have removed the `conffiles` and `control` files from drill root directory pom.xml file https://github.com/apache/drill/compare/c4feef490301f2b9e0f28a1b18dead7117cc3da8..28e617cc8b556e4adadb81379a31cdaa72a13d56 let me know if I am missing anything.

---

@vdiravka I have also removed SNAPSHOT from distribution/pom.xml file hence how come it can generate `apache-drill-1.16.0-SNAPSHOT20190228134219.noarch.rpm` ?  Are you reviewing my latest patch ? You can see the difference https://github.com/apache/drill/compare/c4feef490301f2b9e0f28a1b18dead7117cc3da8..28e617cc8b556e4adadb81379a31cdaa72a13d56

---

* There is no 1.16.0 branch, the last Drill release is 1.15.0. But current Drill [master](https://github.com/apache/drill) branch has [1.16.0-SNAPSHOT](https://github.com/apache/drill/blob/master/pom.xml#L33) version
* No, you didn't remove these files. You removed only their mentions in Drill Maven plugins. This is your branch with this files: https://github.com/nareshgbhat/drill/tree/DRILL-7042/distribution/src/deb/control
* You will have `rpm` with this name after rebase onto Drill master branch. And you are right `<release>SNAPSHOT</release>` is redundant.

So please:
1. rebase your branch onto Drill master branch and force push your changes 
2. remove `conffiles` and `control` files 
3. update `<license>` content

---

@vdiravka  DONE.

---

But it isn't done. 
`<license>http://www.apache.org/licenses/LICENSE-2.0</license>` -> `
<license>${project.organization} ${project.licenses}</license>` as I suggested in the first comment.

And `conffiles` and `control` files are not deleted. They still present in your branch. This is the reason why TravisCI build is failed under your branch: https://travis-ci.org/apache/drill/builds/500336628?utm_source=github_status&utm_medium=notification
the checks for `maven-license-plugin` is removed, but files are not deleted.
Please fix above two issues and then PR can be merged. Or if you want I can fix it in the separate PR, but with your commit.